src/index.js

```diff
@@ -3,10 +3,105 @@ import header from './view/header.js';
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].main === 'Clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1581788945370-3fe92ade72db?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'broken clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1474218861938-d6b14818c8e2?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1489&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'scattered clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1543226862-39202f29696f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].main === 'Rain') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1428592953211-077101b2021b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'shower rain') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1519692933481-e162a57d6721?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'thunderstorm') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1472145246862-b24cf25c4a36?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1351&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'snow') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1465220183275-1faa863377e3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'mist') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1438803235109-d737bc3129ec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=666&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+    })
```

Comment:
[OPTIONAL] You could make a Switch statement for that and put it in a separate module, that would be more concise.

Reply:
Changed if to Switch statement resolved